src/docs/respectful-code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Respectful code"
+description: "Inclusivity is central to V8’s culture, and our values include treating each other with dignity. As such, it’s important that everyone can contribute without facing the harmful effects of bias and discrimination."
+---
+
+Inclusivity is central to V8's culture, and our values include treating each other with dignity. As such, it’s important that everyone can contribute without facing the harmful effects of bias and discrimination. However, terms in our codebase, UIs, and documentation can perpetuate that discrimination. This document sets forth guidance which aims to address disrespectful terminology in code and documentation.
+
+## Policy
+
+Terminology that is derogatory, hurtful, or perpetuates discrimination, either directly or indirectly, should be avoided.
+
+## What is in scope for this policy?
+
+Anything that a contributor would read while working on V8, including:
+
+- Names of variables, types, functions, files, build rules, binaries, exported variables, ...
+- Test data
+- System output and displays
+- Documentation (both inside and outside of source files)
+- Commit messages
+
+## Principles
+
+- Be respectful: Derogatory language shouldn’t be necessary to describe how things work.
+- Respect culturally sensitive language: Some words may carry significant historical or political meanings. Please be mindful of this and use alternatives.
+
+## How do I know if particular terminology is OK or not?
+
+Apply the principles above. If you have any questions, you can reach out to `v8-dev@googlegroups.com`.
+
+## What are examples of terminology to be avoided?
+
+This list is NOT meant to be comprehensive. It contains a few examples that people have run into frequently.
+
+| Term      | Suggested alternatives                                        |
```

Comment:
This needs a `:::table-wrapper`

Reply:
Can you make a suggestion? I have never seen this before and have no idea where to put it. It looked fine when I build it 🤷

Reply:
There are a few examples in the repository — just search for `:::table-wrapper`. It's needed to make the tables look good on narrow viewports. (Ideally we'd get rid of `:::table-wrapper` and instead auto-add it when converting the Markdown, but no one has done that work yet)

Reply:
```suggestion
:::table-wrapper
| Term      | Suggested alternatives                                        |
```

Reply:
You also need the `:::` at the end